modules/camera/config.py

```diff
@@ -1,4 +1,8 @@
+import sys
+
 def can_build(env, platform):
+    if sys.platform.startswith("freebsd"):
```

Comment:
We should probably harmonize how we check for FreeBSD, seems we use either this one or `platform.system() == "FreeBSD"`:
```
SConstruct
341:        or sys.platform.startswith("freebsd")

platform/linuxbsd/detect.py
502:    if platform.system() == "FreeBSD":
508:        if env["use_llvm"] and platform.system() != "FreeBSD":
511:        if env["use_llvm"] and platform.system() != "FreeBSD":
```